Libro.php

```diff
@@ -0,0 +1,32 @@
+<?php
+require_once 'Autor.php';
```

Comment:
```suggestion
require_once "Autor.php";
```
Usar un solo tipo de comillas en el proyecto, simples o dobles